### DISCUSSION TITLE
Effects of switching relevant to irrelevant label during active screening phase (Phase B of SAFE Procedure)

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1823] [Q&A] [answered]
Dear all,

I have a question regarding the effects of changing a screening decision from 'relevant' to 'irrelevant' during an active screening phase. We are in Phase B of the SAFE procedure and have started the Phase B title/abstract screening of a large dataset, and we decided that we want to add specific systematic 'notes' to the included articles. This will be helpful for us later in the full-text screening phase.
Because we have already started screening some abstracts, I was going through the abstracts that I marked as 'relevant' to add these systematic notes to them. While doing this, I came across a few abstracts that I re-read and I thought that they should actually be excluded now that I read it a second time. 
We are using Naive Bayes as the classifier and TF-IDF as the feature extraction technique. Given that we are still in the screening phase, would you advise that I switch the labels of these abstracts from relevant to irrelevant so that the alteration of the next asreview prediction model includes these as irrelevant and not relevant abstracts? Or do you think it is better to stay consistent, and to not switch the labels of these abstracts, given that this was the decision I made at the time? My tendency is to switch the labels, but I wanted to hear your point of view and discuss whether I am messing up the algorithm by doing that. I look forward to reading your input! Many thanks in advance!

Best wishes!

**Comment by Rensvandeschoot** [comment, marked as answer]
I would recommend adjusting the labels to make sure the next iteration of the model has the best information possible. As explained in [this paper](https://doi.org/10.3390/app14093842) a model is trained on the available labels and is not updated but trained from scratch.

**Reply by Morle2104** [reply]
Many thanks! I just read the paper and it was really helpful - thank you! 
If I understand correctly, the changed labelling decision of records is saved automatically by asreview in the .asreview project file and does not need to be manually recorded by the researcher? Best wishes.

**Reply by Rensvandeschoot** [reply]
Exactly! Whether a records label is switched is stored in the project file. We aim for a high level of transparency and reproducibility.

**Reply by Morle2104** [reply]
Perfect, that's great! Many thanks for the quick responses, this really helped!
